sailfishusermanager.cpp

```diff
@@ -211,6 +211,20 @@ uint SailfishUserManager::addUser(const QString &name)
         return 0;
     }
 
+    int count = 0;
+    struct group *grent = getgrnam(USER_GROUP);
+    if (grent)
```

Comment:
Nit: it would be easier to read IMO if this outer check and that for loop inside both would have their own blocks, so basically:
```
    if (grent) {
        for (int i = 0; grent->gr_mem[i]; i++) {
            if (getpwnam(grent->gr_mem[i]))
                count++;
        }
    }
```